index.html

```diff
@@ -291,6 +291,31 @@ <h2>About me</h2>
       </div>
     </main>
 
+    <footer>
+      <div class="form-container">
```

Comment:
[OPTIONAL]

- [ ] I strongly suggest you use the `footer` only for:
- Copyright data
- Links
- Contact information
Since as a rule `footer` is mainly for holding date about the website, adding form into it may easily confuse people with visual disabilities, who uses screen readers to navigate through the web page

- [ ] as well as adding `form-container` to the `main` part of the page, and please kindly prefer using [Semantic HTML5 tags](https://developer.mozilla.org/en-US/docs/Glossary/Semantics) in order to improve the profile in terms of `accessibility`. In this particular case, the `section` tag can be a perfect match instead of `div`